composer.json

```diff
@@ -11,7 +11,8 @@
     "symfony/config": "^3.0||^4.0||^5.0",
     "symfony/dependency-injection": "^3.0||^4.0|^5.0",
     "symfony/http-kernel": "^3.0||^4.0||^5.0",
-    "symfony/messenger": "^4.4||^5.0"
+    "symfony/messenger": "^4.4||^5.0",
+    "friendsofphp/php-cs-fixer": "^2.16"
```

Comment:
Can you move the CS fixer to require-dev please?

Reply:
done.